src/bluesearch/database/mining_cache.py

```diff
@@ -307,7 +307,7 @@ def construct(self):
 
     def _delete_rows(self):
         """Delete rows in the target table that will be re-populated."""
-        for model_id, model_schema in self.model_schemas.items():
+        for model_id, _model_schema in self.model_schemas.items():
```

Comment:
Wow, why?

Reply:
I am interested to know too :) Is it because it is not used in the `for` loop ? If yes, maybe we could just replace by `_` instead ?

Reply:
Yes, exactly, it's because this variable isn't used in the loop body. This was reported by `flake8-bugbear`. I think it's legit - it could be that you're looping over something but due to a typo use a different variable than that you're looping over.

I don't mind using `_` either, to me the only difference might be the readability that the underscored name still let's you understand what kind of objects are being iterated over and why they're not used. But otherwise I don't really mind. What do you guys prefer?

Reply:
Keeping track of the variable name is not a bad idea! But I don't have a strong opinion about this!

Reply:
I am surprised that this is not caught by vanilla `flake8` - after all it is an unused variable, no?

Reply:
that's true, don't know why flake8 doens't spot it by itself